shared/src/main/java/net/blay09/mods/balm/api/event/LevelEvent.java

```diff
@@ -0,0 +1,30 @@
+package net.blay09.mods.balm.api.event;
+
+import net.minecraft.world.level.ChunkPos;
+import net.minecraft.world.level.LevelAccessor;
+import net.minecraft.world.level.chunk.ChunkAccess;
+
+public abstract class LevelEvent {
```

Comment:
LevelEvent clashes with a vanilla class name and I think it's better to stay away from super generic names and have more "dedicated" events instead, since there's so many things that could fall under a "level event" and blow the class up. 

Maybe could call it `LevelLoadingEvent.Load/Unload` or just two seperate `LevelLoadedEvent`/`LevelUnloadedEvent` instead.